test-documents/src/main/resources/documents/tikaConfig.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
If possible, I'd put this in another place. Because `documents` folder should only contain "binary" documents that are going to be sent to FSCrawler when running unit tests or integrations tests.
`tikaConfig.xml` should not be considered as a test document.

I'd recommend keeping this file close to the tests themselves, even if you need to duplicate this file.
Or, you could create another dir named `test-documents/src/main/resources/config` in which you can put the `tikaConfig.xml` file. 

I believe that in the future we might want to have more and more custom Tika config tests.

WDYT?